create-job-ui/src/components/MultiSelectDropDown.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react'
+import Multiselect from 'multiselect-react-dropdown';
+
+
+export const MultiSelectDropDown = ({fetchedOptions,selected,setSelected}) => {
+
+
+    // const[multiSelectOptions,setMultiSelectOptions] = useState(['Contribute in all phases of the development lifecycle',
+    // 'Write well designed, testable, efficient code',
+    // 'Ensure designs are in compliance with specifications']);
+
+    //const[selected,setSelected] = useState([]);
+
+
+    function handleSelect(selectedOption){
+        setSelected([selectedOption]);
+    }
+
+    function handleRemove( selectedList, removedItem){
+     
+        setSelected([selectedList]);
+       
+    }
+
+
+  return (
+    <div>
+
+<Multiselect
```

Comment:
@asharMehmood2211004KHICND good work for wrapping this library in a new component - it means that if we want to change the library later, we only have to do it in one place